open_instruct/grpo_fast.py

```diff
@@ -1061,7 +1061,7 @@ def generate_with_engines(prompts: List[List[int]], sampling_params: SamplingPar
         inference_results_Q.put((response_ids, finish_reasons))
 
         # Evaluate the model
-        if eval_prompt_token_ids is not None and (training_step - 1) % eval_freq == 0:
+        if eval_prompt_token_ids is not None and training_step % eval_freq == 0:
```

Comment:
This seems problematic. Then the initial policy's response would not be logged. Could you revert the change?